profiles/amity-ak97.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
+    "issueId": 1177,
+    "name": "Amit Kumar Yadav",
+    "username": "amity-ak97"
```

Comment:
It looks like there's a 4-space indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
  "issueId": 1177,
  "name": "Amit Kumar Yadav",
  "username": "amity-ak97"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient